### DISCUSSION TITLE
Tab bar color and terminal prompt color don't match the background

### DISCUSSION THREAD

**Discussion by rsmath** [kovidgoyal/kitty#5642] [Q&A]
Hello.

I recently changed the background color of my Kitty terminal. But in the top-left (where ramanshsharma@Jarvis is) and the bottom tab bar, the colors are still not changed. Does anyone know which parameter value I have to change in kitty.conf for this?

Thank you.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/24642071/199577601-91ff9aa5-5f75-4f06-a18c-d4e293924c17.png">

**Comment by page-down** [comment]
> ... prompt color ...

Change the color of the prompt configured in your shell rc or associated configuration file.
As for where exactly to change, it depends on what you are using. You will need to find it yourself or ask in the respective project community.

Many times the prompt uses the colors specified by the prompt theme, so the color values in `kitty.conf` will not be used by the program (shell).

> ... the bottom tab bar ...

See the configuration options related to tab colors in the following link.
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.active_tab_foreground

If you are using a custom tab style, modify the corresponding python script.
